.github/workflows/make_wheel_macOS_x86.sh

```diff
@@ -16,8 +16,8 @@ bazel clean
 python configure.py
 
 bazel build \
-  --copt -mmacosx-version-min=10.13 \
-  --linkopt -mmacosx-version-min=10.13 \
+  --copt -mmacosx-version-min=10.14 \
+  --linkopt -mmacosx-version-min=10.14 \
```

Comment:
I think there might be a `=` missing here. Just wondering why the platform name [currently is `10.15`](https://github.com/tensorflow/addons/pull/2744/files#diff-c0354faba8e95ccf3419158e87d1d2f027b38a82d4bf88b14305e9f9f69d077dL83) which makes me think that this option might be silently ignored.
```suggestion
  --copt=-mmacosx-version-min=10.14 \
  --linkopt=-mmacosx-version-min=10.14 \
```

Reply:
I believe we were building 10.13 and labeling 10.15 with delocate

Reply:
> It would be nice if @jhavukainen or @kulinseth could clarify on this.
@bhack , what's the question ?

Reply:
https://github.com/tensorflow/addons/pull/2744#issuecomment-1252165641

Reply:
@kulinseth see the link above